### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by clemens-fricke**
Multiple smallish commits fixing some spelling and style guide issues.

This is now more in line to follow the pep8 style guide. 

In reference to #105

### CODE REVIEW

**Review by j042 — COMMENTED** [review]
Nice clean up, thanks! One question, one suggestion.
@jzwar, would this cause too much conflict for microstructures that you are working on? Iff so, we can takeout microstructures for this PR.

File: gustaf/spline/microstructure/tiles/__init__.py [hunk 1]
```diff
@@ -2,13 +2,6 @@
 
 Interface for tools and generators creating simple microstructures.
 """
-
-from gustaf.spline.microstructure.tiles import (
```
[the remark this answers is not in the document]
Comment (clemens-fricke) [reply gustaf/spline/microstructure/tiles/__init__.py:6]: Als ich das gemacht habe hatte ich den nutzten nicht wirklich gesehen, aber inzwischen verstehe ich warum das auch nochmal extra genannt wird.

**Comment by clemens-fricke** [comment]
The requested changes are incorporated. I reverted the changes in the microstructure folder due to possible conflicts with the currently open microstructure PR.

**Comment by j042** [comment]
I just took out 3 commits related to microtiles. @clemens-fricke could you please double check if everything is ok?

**Comment by j042** [comment]
@jzwar could you please check if there's any conflict with #95 ?

**Comment by jzwar** [comment]
> @jzwar could you please check if there's any conflict with #95 ?

No conflicts with ft-microstructure-derivatives #95

**Review by j042 — APPROVED** [review]
Thanks again!

**Pull request merged**